gui/app/(tabs)/login.tsx

```diff
@@ -0,0 +1,108 @@
+// Login and SignUp page 
+
+import React, { useState } from 'react';
+import { StyleSheet, View, TextInput, Button, Text } from 'react-native';
+// Login Page
+const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
```

Comment:
We can issue a remember token, caching the plain text password in the state probably isn't a great thing to do.

Reply:
How can i get the token from the login API?

Reply:
Will be covered in #14 , no need to worry about it here.